sdcm/logcollector.py

```diff
@@ -624,8 +624,10 @@ def collect_logs_per_node(node):
                 except Exception as details:  # pylint: disable=unused-variable, broad-except
                     LOGGER.error("Error occured during collecting on host: %s\n%s", node.name, details)
 
+        LOGGER.info("Nodes list %s", [node.name for node in self.nodes])
+
         if not self.nodes:
-            LOGGER.warning(f'No nodes found for in {self.cluster_log_type} cluster')
+            LOGGER.error(f'No nodes found for {self.cluster_log_type} cluster. Logs will not be collected')
```

Comment:
this is clearly a `warning` log level, not errror.
Common definition of warning: 
```
Potentially harmful situations of interest to end-users or system managers that indicate potential problems.
```
An error means that an **error** happened, in the source code execution, but we know how to handle it and program can continue its execution.

Reply:
fixed